src/PhpDocument.php

```diff
@@ -284,11 +247,21 @@ public function getNodeAtPosition(Position $position)
         if ($this->stmts === null) {
             return null;
         }
-        $traverser = new NodeTraverser;
-        $finder = new NodeAtPositionFinder($position);
-        $traverser->addVisitor($finder);
-        $traverser->traverse($this->stmts);
-        return $finder->node;
+
+        if (\is_array($this->stmts)) {
```

Comment:
From what I understand, this is actually more performant because PHP will know that it's the global `is_array`.